extensions/terminal-suggest/src/completions/code.ts

```diff
@@ -332,3 +347,16 @@ const codeCompletionSpec: Fig.Spec = {
 };
 
 export default codeCompletionSpec;
+
+
+export function parseInstalledExtensions(out: string): Fig.Suggestion[] | undefined {
+	const extensions = out.split('\n').filter(Boolean).map((line) => {
+		const [id, version] = line.split('@');
+		return {
+			name: id,
+			type: 'option' as Fig.SuggestionType,
+			description: `Version: ${version}`
+		};
+	});
+	return extensions;
+}
```

Comment:
We'll want this put a bit higher so the export is the last line, I think this is what all the other specs do.